cluster/gce/util.sh

```diff
@@ -898,7 +907,21 @@ REGION: $(yaml-quote ${REGION})
 VOLUME_PLUGIN_DIR: $(yaml-quote ${VOLUME_PLUGIN_DIR})
 KUBELET_ARGS: $(yaml-quote ${KUBELET_ARGS})
 REQUIRE_METADATA_KUBELET_CONFIG_FILE: $(yaml-quote true)
+ENABLE_NETD: $(yaml-quote ${ENABLE_NETD:-false})
+CUSTOM_NETD_YAML: |
+$(echo "${CUSTOM_NETD_YAML:-}" | sed -e "s/'/''/g")
```

Comment:
@jingax10 We have to do this, because the python yaml lib has a bug, it will detect the "---" and raise an error even it is in single quoted. 
https://github.com/yaml/pyyaml/blob/master/lib/yaml/scanner.py#L1264

Reply:
why we need CUSTOM_NETD_YAML here? Please refer to an example of CUSTOM_KUBE_DNS_YAML.

Reply:
I meant you should remove CUSTOM_NETD_YAML. Just take customized DNS as example, the relevant CUSTOM_KUBE_DNS_YAML is not needed here.